src/main/kotlin/systems/ajax/malov/stockanalyzer/repository/StockRepository.kt

```diff
@@ -3,7 +3,7 @@ package systems.ajax.malov.stockanalyzer.repository
 import systems.ajax.malov.stockanalyzer.entity.Stock
 
 interface StockRepository {
-    fun insertAll(stocks: List<Stock>): MutableCollection<Stock>
-    fun findFiveBestStocksToBuy(): List<Pair<String?, List<Stock>>>
-    fun getAllStockSymbols(): List<String?>
+    fun insertAll(stocks: List<Stock>): List<Stock>
+    fun findTopNStocks(n: Int): List<Pair<String?, List<Stock>>>
```

Comment:
at the demo we discussed that this method can return just List<Stock>. is there a reason you didn't change it?

Reply:
This function cannot just return a List<Stocks> due to business logic. The method name isn't good. So I will change it